Components/Button/button.css

```diff
@@ -0,0 +1,411 @@
+.button-container
+{
+    width: 100%;
+    height: fit-content;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 3rem 1rem;
+}
+
+
+/* Primary Solid Buttons */
+.solid-primary-btn
+{
+    min-width: 120px;
+    min-height: 45px;
+    padding: 10px;
+    margin: 5px;
+    border: none;
+    border-radius: 4px;
+    font-family: Comfortaa;
+    color: var(--background-color);
+    background-color: var(--primary-dark-blue-color);
+}
+
+.solid-primary-btn
+{
+    background-position: center;
+    transition: background 0.5s;
+}
+
+.solid-primary-btn:hover
+{
+    background: #514f8b
+                radial-gradient(circle, transparent 1%, 
+                                    var(--primary-dark-blue-color) 1%) center/15000%;
+}
+
+.solid-primary-btn:active 
+{
+    background-color: var(--primary-dark-blue-color);
+    background-size: 100%;
+    transition: background 0s;
+}
+
+
+
+.solid-secondary-btn
+{
+    min-width: 120px;
+    min-height: 45px;
+    padding: 10px;
+    margin: 5px;
+    border: none;
+    border-radius: 4px;
+    font-family: Comfortaa;
+    color: var(--background-color);
+    background-color: var(--secondary-pink-color);
+}
+
+.solid-secondary-btn 
+{
+    background-position: center;
+    transition: background 0.5s;
+}
+
+.solid-secondary-btn:hover 
+{
+    background: #db6b8b
+                radial-gradient(circle, transparent 1%, 
+                                    var(--secondary-pink-color) 1%) center/15000%;
+}
+
+.solid-secondary-btn:active 
+{
+    background-color: var(--secondary-pink-color);
+    background-size: 100%;
+    transition: background 0s;
+}
+
+
+
+
+.solid-error-btn
+{
+    min-width: 120px;
+    min-height: 45px;
+    padding: 10px;
+    margin: 5px;
+    border: none;
+    border-radius: 4px;
+    font-family: Comfortaa;
+    color: var(--background-color);
+    background-color: var(--notification-or-error);
+}
+
+.solid-error-btn 
+{
+    background-position: center;
+    transition: background 0.5s;
+}
+
+.solid-error-btn:hover 
+{
+    background: #ff5f53
+                radial-gradient(circle, transparent 1%, 
+                                    var(--notification-or-error) 1%) center/15000%;
+}
+
+.solid-error-btn:active 
+{
+    background-color: var(--notification-or-error);
+    background-size: 100%;
+    transition: background 0s;
```

Comment:
use of this?

Reply:
This one is needed for button click ripple animation, if I remove it then animation won't work and if I increase its time, animation will work way too late after button click, I wanted animation to start exactly on button click.